### ISSUE TITLE
Support for https://github.com/c9/azure-packager-node

### ISSUE THREAD

**Issue by johnnyhalife** [tjanczuk/git-azure#52]
This way we don't need Windows for repackaging the whole thing (if needed). The idea is to provide an alternative for `build.cmd` to create the package on non Windows environments

**Comment by tjanczuk** [comment]
Yes, we should look into this. Revving the cspks is both inconvenient and pollutes the repo. Plus the problem will be more apparent with multiple VM sizes.

**Comment by johnnyhalife** [comment]
Definitely, and then our only dependency will be with download.exe, we can either bundle curl or download.exe, and we'll have a 100% node based tool.

Things like VM Sizes, Config Vars, etc could be changed and built by the time we do the init...

Powerful.
